si/orange3.jaml

```diff
@@ -2414,7 +2414,7 @@ modelling/base.py:
 modelling/catgb.py:
     CatGBLearner: false
     class `CatGBLearner`:
-        Gradient Boosting (catboost): false
+        Gradient Boosting (catboost): Gradientno povečanje (catboost)
```

Comment:
Za tale prevod (boosting -> povečanje) nisem prepričan. Mogoče za vsak slučaj vprašaš @markotoplak in @BlazZupan?